src/PayU/Alu/Response.php

```diff
@@ -119,6 +119,44 @@ class Response
      */
     private $type;
 
+    /** @var int */
+    private $tokenCode;
+
+    /** @var string */
+    private $tokenMessage;
+
+    /**
+     * @return int
+     */
+    public function getTokenCode()
```

Comment:
Since they always go together, maybe we could extract tokenCode and tokenMessage in a separate object and here in response have a property tokenData or something

Reply:
done